resources/config.rb

```diff
@@ -11,11 +11,33 @@
 property :path, String, name_property: true
 property :settings, Hash, default: {}
 
+def merge_args(arghash)
+    out = []
```

Comment:
Lint nitpick: this should be a two-space indent, not a 4-space one.

Reply:
oops